Tests/test_file_jpeg2k.py

```diff
@@ -29,8 +30,16 @@
 
 pytestmark = skip_unless_feature("jpg_2000")
 
-test_card = Image.open("Tests/images/test-card.png")
-test_card.load()
+
+@pytest.fixture
+def test_card() -> Generator[ImageFile.ImageFile, None, None]:
```

Comment:
Good idea! 

I know the old one was also prefixed `test_`, but that makes it look like a test function. Maybe just `card`? 
```suggestion
def card() -> Generator[ImageFile.ImageFile, None, None]:
```

Reply:
Sure, I've pushed a commit.